server/pbenchinacan/container-build.sh

```diff
@@ -1,24 +1,29 @@
-#!/usr/bin/env bash
+#!/usr/bin/env bash -x
```

Comment:
I am not in favor of keeping `-x`, the verbosity can be over-whelming at the wrong times.

Reply:
This is a typo!  (Um, brain-cramp, really....)

I believe that that should be `-e`, not `-x`.